### ISSUE TITLE
Error: Request failed with status 400 (URL: https://relay.flashbots.net): {"error":"unable to decode txs"}

### ISSUE THREAD

**Issue by design-smith** [flashbots/ethers-provider-flashbots-bundle#127]
I keep getting that error when I try to simulate my transaction. This is the code:

**Comment by Fahadulhassan1** [comment]
flashbotsprovider = await FlashbotsBundleProvider.create(
          provider,
          authSigner,
         "https://wallet relay url"
         "sepolia"
        );
      pass your relay url and network as well. even it's optional but optional is only for mainnet

**Comment by design-smith** [comment]
Thank you so much for your help.
Does this 'https://relay.flashbots.net' work as a wallet relay URL?

**Comment by design-smith** [comment]
I implemented your suggestion and it kinda worked.
Now I am getting a new error "Simulation Error: the method eth_callBundle
does not exist/is not available"
Any idea what might be the problem?

**Comment by Fahadulhassan1** [comment]
yes I faced this issue as well. may be you're passing wrong replay.
